services/nris-api/backend/app/extensions.py

```diff
@@ -10,7 +9,7 @@
 
 apm = ElasticAPM()
 db = SQLAlchemy()
-migrate = Migrate()
+migrate = Migrate(compare_type=True)
```

Comment:
interesting that this is needed. should be default IMO?

Reply:
As far as Jason and I could find yesterday it is not.